.github/workflows/finishReleaseCycle.yml

```diff
@@ -56,13 +56,23 @@ jobs:
     needs: validate
     if: ${{ fromJSON(needs.validate.outputs.isValid) }}
     steps:
-      - name: Update production branch
-        uses: Expensify/App/.github/actions/composite/updateProtectedBranch@main
+      - uses: actions/checkout@v3
+        with:
+          ref: staging
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
+
+      - uses: Expensify/App/.github/actions/composite/setupGitForOSBotify@main
         with:
-          TARGET_BRANCH: production
-          OS_BOTIFY_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           GPG_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
 
+      - name: Update production branch
+        run: |
+          git switch production
```

Comment:
Little note on `git switch` in case you're not familiar:

```
If you want to switch to a remote branch that does not exist as local branch in your local working directory, you can simply execute git switch remoteBranch. When Git is unable to find this branch in your local repository, it will assume that you want to checkout the respective remote branch with the same name. It will then create a local branch with the same name. It will also set up a tracking relationship between your remote and local branch so that git pull and git push will work as intended.
```